NuRadioMC/SignalGen/ARZ/ARZ.py

```diff
@@ -11,9 +11,19 @@
 import os
 import copy
 import logging
+import six
 logger = logging.getLogger("SignalGen.ARZ")
 logging.basicConfig()
 
+
+class Singleton(type):
```

Comment:
I think it would be better to have this class in its own file in the utilities folder, preferably in NuRadioReco. It's not unlikely that it will be used somewhere else in the future, and if someone looks for it it will be hard to find here.

Reply:
I'll change that once your NuRadioReco PR is merged that provides this utility

Reply:
I added a meta class. I anyway had a corresponding NuRadioReco PR